src/tsconfig.cjs.json

```diff
@@ -4,5 +4,8 @@
     "module": "commonjs",
     "target": "es5",
     "outDir": "../dist/cjs"
-  }
+  },
+  "exclude": [
+    "./internal/umd.ts"
```

Comment:
cjs doesn't emit umd entrypoint anyway, but this makes compiler confuses to retry incremental build always (cause incremental build checks build output exists). simply exclude it so incremental build works properly.